src/formBuilder/utils.js

```diff
@@ -879,23 +879,14 @@ export function updateSchemas(
     generateSchemaFromElementProps(elementArr),
   );
 
-  const existingUiSchema: {
+  const newUiSchema: {
     [string]: any,
     definitions?: { [string]: any },
     ...
-  } = Object.entries(uischema)
-    .filter(([key, _value]) => {
-      return key in newSchema.properties || key.startsWith('ui:');
-    })
-    .reduce((accumulator, currentValue) => {
-      const [key, value] = currentValue;
-      return { ...accumulator, [key]: value };
-    }, {});
-
-  const newUiSchema = Object.assign(
```

Comment:
@bradleymarques @cnxumalo
The issue is that Radio has a entry in defaultUiSchema, but Short Answer has a defaultUiSchema of {}. So here, if an input is a Radio, and we then change the input type from Radio to Short Answer, the Object.assign fails to remove the uiSchema entry for when the input was a Radio. Short Answer is supposed to have no uiSchema, but the result here is a Short Answer input with an unexpected uiSchema, causing an error.